### PR TITLE
Update 2.6.0

### PR DESCRIPTION
**Pull request by ryanskeith**
- Update version, hash and dependencies.
- Should build on more platforms now.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
-  url: https://pypi.org/packages/source/g/google-cloud-storage/google-cloud-storage-{{ version }}.tar.gz
-  sha256: f3b4f4be5c8a1b5727a8f7136c94d3bacdd4b7bf11f9553f51ae4c1d876529d3
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 104ca28ae61243b637f2f01455cc8a05e8f15a2a18ced96cb587241cdd3820f5
 
 build:
   number: 0
-  # A lot of dependencies are not available on s390x and win32 currently
-  skip: True  # [py<36 or s390x or ppc64le or win32]
+  # A lot of dependencies are not available on s390x currently  # [py<37 or s390x or ppc64le]
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: As now we can build on `s390x` and `ppc64le` the comment is redundant

File: recipe/meta.yaml [hunk 2]
```diff
@@ -23,12 +24,11 @@ requirements:
   run:
     - python
     - google-auth >=1.25.0,<3.0dev
-    - google-api-core >=1.29.0,<3.0dev
-    - google-cloud-core >=1.6.0,<3.0dev
-    - google-resumable-media >=1.3.0,<3.0dev
+    - google-api-core >=1.31.5,<3.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: For **google-api-core**, should be `3.0.0dev `instead of `3.0dev`

File: recipe/meta.yaml [hunk 3]
```diff
@@ -23,12 +24,11 @@ requirements:
...
+    - google-cloud-core >=2.3.0,<3.0dev
+    - google-resumable-media >=2.3.2
     - requests >=2.18.0,<3.0.0dev
-    - protobuf
-    - six
+    - protobuf <5.0.0dev
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:31]: Protobuf is an extras dependency, https://github.com/googleapis/python-storage/blob/aee58b2b0eeafe3e26090df70951a5688148befb/setup.py#L37. Could you add a comment like conda-forge did? See https://github.com/conda-forge/google-cloud-storage-feedstock/blob/0b29f40b1aff6e92d6e130e0f503c91e7731117e/recipe/meta.yaml#L28

**Review by varlackc — COMMENTED** [review]
As soon as the required changes take place, we will be ready to approve the PR.

**Pull request merged**